src/app/components/Transcript/index.styles.tsx

```diff
@@ -26,28 +26,39 @@ export default {
       color: isDarkUi ? palette.WHITE : palette.GREY_10,
     }),
 
-  ul: ({ spacings, palette, isDarkUi }: Theme) =>
+  ul: ({ spacings, mq }: Theme) =>
     css({
-      paddingTop: `${spacings.DOUBLE}rem`,
-      paddingInlineStart: '0',
+      padding: `0 ${spacings.FULL}rem`,
       listStyle: 'none',
-      li: {
-        paddingBottom: `${spacings.HALF}rem`,
-        // check on hack prototype what these style do
-        // '::before': {
-        //   listStyle: 'none',
-        //   top: '0',
-        //   position: 'absolute',
-        //   borderWidth: '0',
-        //   border: 'none',
-        //   borderRadius: '0',
-        //   left: '0',
-        // },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        padding: `0 ${spacings.DOUBLE}rem`,
       },
     }),
 
-  itemText: ({ spacings }: Theme) =>
+  itemText: ({ spacings, mq }: Theme) =>
     css({
-      paddingInlineStart: `${spacings.FULL}rem`,
+      float: 'left',
+      width: `100%`,
+      [mq.GROUP_1_MIN_WIDTH]: {
+        paddingInlineStart: `${spacings.FULL}rem`,
+        width: `calc(75% - ${spacings.FULL}rem)`,
+      },
+      [mq.GROUP_2_MIN_WIDTH]: {
+        width: `calc(85% - ${spacings.FULL}rem)`,
+      },
+      [mq.GROUP_3_MIN_WIDTH]: {
+        paddingInlineStart: `${spacings.DOUBLE}rem`,
+        width: `calc(90% - ${spacings.DOUBLE}rem)`,
+      },
+    }),
+
+  listItem: ({ spacings }: Theme) =>
+    css({
+      paddingBottom: `${spacings.DOUBLE}rem`,
+      '::after': {
+        content: '""',
+        display: 'table',
```

Comment:
What's this one for @Isabella-Mitchell? This can make a screen reader announce table semanics, we can test it, though we should avoid using this if possible

Reply:
Yep fair -` display: 'block'` seems to work just as well here so I have changed this.

Note this doesn't seem to be ideal CSS since it's a bit hard to read/ understand. This adds spacing underneath each line item. 

Flexbox is a possible alternative.